### PR TITLE
Build new /core/features/secure-boot page

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build new `/core/features/secure-boot` page
- Add to secondary nav

## QA

- View page at: https://ubuntu-com-9119.demos.haus/core/features/secure-boot
- Compare against [copy doc](https://docs.google.com/document/d/1ltH7T2boIJ7VcUni4Kmc0XGlmmxWAf5u82LEwkEGP24/edit#)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Waiting to merge `/core/features` first 

## Issue / Card

Fixes [#3640](https://github.com/canonical-web-and-design/web-squad/issues/3640)

### CODE REVIEW

**Comment by anasereijo** [comment]
- [ ] title missing here:
Screenshot 2021-01-28 at 16 16 28

- [ ] hide contact us image on mobile version

**Comment by bethcollins92** [comment]
@anasereijo done 👍🏼

**Comment by anasereijo** [comment]
@bethcollins92 LGTM

**Comment by bethcollins92** [comment]
@pmahnke ready for review

File: templates/core/features/secure-boot.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+<section class="p-strip--suru-topped">
+  <div class="row u-equal-height">
+    <div class="col-7">
+      <h1>Secure boot</h1>
+      <p class="p-heading--4">Protect against vulnerabilities at boot time</p>
+      <p>Computers are vulnerable during the boot process if they are not secured. The kernel, hardware peripherals and user space processes are all initiated at boot and any vulnerability in the boot firmware can have cascading effects on the entire system.</p>
+      <p>In the case of an attack on boot firmware, damages are so profound that often hardware replacement is the only fix. In an industrial IoT scenario, this means considerable downtime, manual maintenance, possibly at several locations and CapEx for hardware replacement. An utterly undesirable outcome.</p>
+      <p>
+        <a href="/core#get-in-touch" class="p-button--positive js-invoke-modal">Improve security at boot time with Ubuntu CoreSecure your devices</a>
```
Comment (pmahnke) [inline templates/core/features/secure-boot.html:16]:
```suggestion
        <a href="/core/docs/security-and-sandboxing">Learn about security and sandboxing in Ubuntu Core&nbsp;&rsaquo;</a>
```

I made Galem update this CTA.

**Review by pmahnke — APPROVED** [review]
+1 if you fix that CTA

**Pull request merged**